### ISSUE TITLE
Incompatible with 1.10 (chest getCustomName) [4.5.2]

### ISSUE THREAD

**Issue by detobel36** [ConnorLinfoot/CratesPlus#657]
### Debug Info
Useless here (related to code)

### Link/paste of stack trace/error (if any)
```YAML
Caused by: java.lang.NoSuchMethodError: org.bukkit.block.Chest.getCustomName()Ljava/lang/String;
	at plus.crates.Listeners.PlayerInteract.onPlayerInteract(PlayerInteract.java:42) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor77.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_212]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_212]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot.jar:git-Spigot-570526c-75a8885]
```

### Actions to reproduce (if known):
Make an interaction with a Block that doesn't have Metadata "CreateType"

### Code description

In the code we can read this:
https://github.com/ConnorLinfoot/CratesPlus/blob/master/src/main/java/plus/crates/Listeners/PlayerInteract.java#L42
```JAVA
Chest chest = (Chest) event.getClickedBlock().getState();
String title = chest.getCustomName();
```
But they doesn't have any `getCustomName` for the Chest in 1.10           
https://helpch.at/docs/1.10.2/org/bukkit/block/Chest.html

**Comment by DwarfyAssassin** [comment]
Having the same issue on 1.7.10 Thermos.
It doesn't really seem to break things but the holograms don't appear any more.

**Comment by ConnorLinfoot** [comment]
Fixed in upcoming update #793
